Automattic/Gravatar-SDK-iOS
# Adding customURL option to DefaultAvatarOption

Closes #152 

### Description

Adding a custom URL option to DefaultAvatarOption.

### Testing Steps

Using the Gravatar Demo app:

- Go to `ImageDownload - Networking` page
- Enter an invalid string as `Email`
- Search on internet any image and get its URL
- Paste the URL on the field `Set custom avatar URL`
- Press Enter to save the change
- Check that the Default Avatar Option now says `Custom URL` on the button abov.
- Tap on `Fetch Avatar`.
  - Check that the image from the URL is displayed
